frame/ethereum/src/tests/eip1559.rs

```diff
@@ -411,6 +412,91 @@ fn call_should_handle_errors() {
 	});
 }
 
+#[test]
+fn event_extra_data_should_be_handle_properly() {
+	// 	pragma solidity ^0.6.6;
+	// 	contract Test {
+	// 		function foo() external pure returns (bool) {
+	// 			return true;
+	// 		}
+	// 		function bar() external pure {
+	// 			require(false, "error_msg");
```

Comment:
Ideally we want to test that no matter how long the revert message is, we have a boundary for it or the test will fail if `ExtraDataLength` changes its behaviour or is removed.

```suggestion
	// 			require(false, "very_long_error_msg_that_we_expect_to_be_trimmed_away");
```

Reply:
Make sense, fixed.